modules/ept-frontend/client/app.js

```diff
@@ -75,10 +75,7 @@ require('./patrol/resource');
 // Set Angular Configs
 app
 .config(require('./config'))
-.run(['$rootScope', '$state', '$timeout', 'Alert', 'BreadcrumbSvc', 'BanSvc', 'Websocket', function($rootScope, $state, $timeout, Alert, BreadcrumbSvc, BanSvc, Websocket) {
-
-  // start the websocket connection on page load
-  Websocket.init();
```

Comment:
are you sure this is the only place it is called?

Reply:
I think so.

Reply:
can you please be certain?  i can't approve of this change unless you can say for sure.

Reply:
can you please be certain? i can't approve of this change unless you can say for sure.

Reply:
it is

Reply:
okay